esx_service/vmdk-opsd

```diff
@@ -150,7 +184,7 @@ check_running_status () {
    fi
 
    echo "${OPSD_TAG} is not running"
-   return 6
+   return 5
```

Comment:
Why change the error code from "6" to "5"?

Reply:
Here's the previous error codes:
```
1 - failed to start (already running)
2 - failed to create resource pool
3 - failed to start (error happens)
4 - failed to delete resource pool
5 - failed to stop vmdkops
6 - vmdkops not running
```
After introducing 'shutdown' option, we need a new error code for "failed to shut down vmdkops". I can use 7 for that, but I feel we should use two consecutive code for these 2 error conditions, so I changed it like this:
```
1 - 4 (same as above)
5 - vmdkops not running
6 - failed to stop vmdkops
7 - failed to shut down vmdkops
```
I don't think these error code are really being used anywhere. If there are concerns about backward compatibility, I can change it back.

Reply:
FWIW I have no concerns, I just did not see it matters so I was surprised by the change :-)